packages/web/components/cards/pool-card.tsx

```diff
@@ -0,0 +1,197 @@
+import { ObservablePool } from "@osmosis-labs/stores";
+import { observer } from "mobx-react-lite";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { FunctionComponent } from "react";
+import { useStore } from "../../stores";
+import { useDeterministicIntegerFromString } from "../../hooks";
+import classNames from "classnames";
+
+const poolCardIconBackgroundColorsToTailwindBgImage = {
+  socialLive: "bg-gradients-socialLive",
+  greenBeach: "bg-gradients-greenBeach",
+  kashmir: "bg-gradients-kashmir",
+  frost: "bg-gradients-frost",
+  cherry: "bg-gradients-cherry",
+  sunset: "bg-gradients-sunset",
+  orangeCoral: "bg-gradients-orangeCoral",
+  pinky: "bg-gradients-pinky",
+} as const;
+
+const PoolCardIconBackgroundColors: (keyof typeof poolCardIconBackgroundColorsToTailwindBgImage)[] =
+  [
+    "socialLive",
+    "greenBeach",
+    "kashmir",
+    "frost",
+    "cherry",
+    "sunset",
+    "orangeCoral",
+    "pinky",
+  ];
+
+export const PoolCard: FunctionComponent<{
+  pool: ObservablePool;
+  isMyPool: boolean;
+}> = observer(({ pool, isMyPool }) => {
+  const { chainStore, queriesOsmosisStore, priceStore, accountStore } =
+    useStore();
+  const router = useRouter();
+  const deterministicInteger = useDeterministicIntegerFromString(pool.id);
+  const chainInfo = chainStore.osmosis;
+  const accountInfo = accountStore.getAccount(chainStore.osmosis.chainId);
+  const queryOsmosis = queriesOsmosisStore.get(chainInfo.chainId);
+
+  const apr = queryOsmosis.queryIncentivizedPools.computeMostAPY(
+    pool.id,
+    priceStore,
+    priceStore.getFiatCurrency("usd")!
+  );
+
+  const poolLiquidity = pool.computeTotalValueLocked(
+    priceStore,
+    priceStore.getFiatCurrency("usd")!
+  );
+
+  const bondedShareRatio =
+    queryOsmosis.queryGammPoolShare.getLockedGammShareRatio(
+      accountInfo.bech32Address,
+      pool.id
+    );
+  const bonded = poolLiquidity
+    .mul(bondedShareRatio.moveDecimalPointLeft(2))
```

Comment:
Everything above this looks fine (getting data), but I see some opportunity here to put this math logic into `ObservableQueryGammPoolShare`. Perhaps alongside `getLockedGammShareRatio()`.